swan-lake/integration-tools/health-tool.md

```diff
@@ -1,90 +1,150 @@
 ---
 layout: ballerina-health-support-left-nav-pages-swanlake
 title: Health tool (FHIR/HL7)
-description: The health tool generates API templates and library packages for developing healthcare integrations.
-keywords: ballerina, programming language, health, contract
+description: The health tool generates Ballerina library packages and Ballerina templates for FHIR APIs for developing healthcare integrations.
+keywords: ballerina, programming language, healthcare, contract
 permalink: /learn/health-tool/
 active: health-tool
-intro: The health tool generates API templates and library packages for developing healthcare integrations.
+intro: The health tool generates Ballerina library packages and Ballerina templates for FHIR APIs for developing healthcare integrations.
 ---
 
-They are primarily focused on the [Fast Healthcare Interoperability Resources (FHIR)](https://www.hl7.org/fhir/overview.html) standard and are developed using Ballerina.
+These artifacts generated by the tool are primarily focused on the [Fast Healthcare Interoperability Resources (FHIR)](https://www.hl7.org/fhir/overview.html) standard and are developed using Ballerina.
```

Comment:
I don't think this is still clear when we start with `These`. We need to be clear about what is meant by `These`

Reply:
The para starts like this `The health tool generates Ballerina library packages and Ballerina templates for FHIR APIs for developing healthcare integrations. These artifacts generated by the tool are primarily focused on the ...`. So by `These` we mean the generated bal pkgs and templates.

Reply:
I see, intro line comes first. Sounds good.